x-pack/plugins/security_solution/public/detections/components/no_write_alerts_callout/translations.ts

```diff
@@ -21,6 +21,7 @@ export const NO_WRITE_ALERTS_CALLOUT_MSG = i18n.translate(
   }
 );
 
+// TODO: del
 export const DISMISS_CALLOUT = i18n.translate(
   'xpack.securitySolution.detectionEngine.dismissNoWriteAlertButton',
```

Comment:
What happens when/if I delete it?
What happens when I add a new localization dictionary key?

Reply:
There's an [internationalization check script](https://github.com/elastic/kibana/blob/master/scripts/i18n_check.js) that runs in the CI that will throw an error if there's a translation that no longer has a usage in the code. I believe that's based on that key so changing it would have the same effect

Reply:
👍  Thank you. Yep, I adjusted some translation keys, run the script and it did blame on me, so I tweaked the translation to make it happy again.